### DISCUSSION TITLE
custom events suddenly stopped  working in android only

### DISCUSSION THREAD

**Discussion by abdRida** [invertase/react-native-firebase#7185] [Q&A]
Why have my custom events suddenly stopped appearing in Firebase debug mode for Android in React Native, while they continue to work fine on iOS? Previously, everything was functioning correctly, but now the custom events are not showing up for Android. What could be causing this unexpected change?

I am using @react-native-firebase/analytics

analytics().logEvent("test")

Only the "screen_view" event is successfully appearing in the Firebase console when logged, but any other events are not being displayed. For example, if I add a button to the screen and log the "screen_view" event using analytics().logEvent("screen_view"), when I click the button multiple times and check the events in the debug view of the Firebase console, the "screen_view" event appears multiple times as expected. However, when I log any other custom events ("test etc...), they do not appear in the Firebase console's debug view.

**Comment by dancherb** [comment]
Hi, did you resolve this? I've recently installed Firebase in an Expo project and seem to be having the same issue.
